### PR TITLE
[macOS] Add Java to the macOS 13 arm64

### PR DESCRIPTION
**Pull request by Alexey-Ayupov**
# Description
 Add Java to the macOS 13 arm64

#### Related issue:
https://github.com/actions/runner-images-internal/issues/5447

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/macos/tests/Java.Tests.ps1 [hunk 1]
```diff
@@ -59,16 +64,20 @@ Describe "Java" -Skip:($os.IsVenturaArm64) {
...
-        
-            It "Gradle is installed to /usr/local/bin" {
+
+            It "Gradle is installed to /usr/local/bin" -Skip:($os.IsVenturaArm64)  {
                 (Get-Command "gradle").Path | Should -BeExactly "/usr/local/bin/gradle"
             }
+
+            It "Gradle is installed to /opt/homebrew/bin/gradle" -Skip:($os.IsVentura)  {
+                (Get-Command "gradle").Path | Should -BeExactly "/opt/homebrew/bin/gradle"
+            }
```
Comment (vpolikarpov-akvelon) [inline images/macos/tests/Java.Tests.ps1:80]: Won't these two test cases interfere in case of Mac OS 11 or 12?

**Review by mikhailkoliada — APPROVED** [review]

**Pull request merged**